include/oneapi/dpl/pstl/iterator_impl.h

```diff
@@ -940,6 +1008,16 @@ namespace dpl
 {
 namespace __internal
 {
+template <typename _T>
+struct __is_reversed_indirectly_device_accessible_it : std::false_type
+{
+};
+
+template <typename _BaseIter>
+struct __is_reversed_indirectly_device_accessible_it<std::reverse_iterator<_BaseIter>>
```

Comment:
Do we support `std::move_iterator`? If so, do we need a specialization similar to this one for `std::reverse_iterator`?